packages/ui/src/components/sign-in/steps/choose-session.tsx

```diff
@@ -0,0 +1,162 @@
+            {activeSessions?.map(session => {
+              const { userId, identifier, firstName, lastName, hasImage, imageUrl } = session.publicUserData;
+              let title = '';
+              let subtitle = '';
+              if (firstName || lastName) {
+                title = `${firstName} ${lastName}`;
+                subtitle = identifier;
+              } else {
+                title = identifier;
+              }
+              return (
+                <li
+                  key={userId}
+                  className='border-gray-a4 border-t'
+                >
+                  <Button
+                    type='button'
+                    className={sessionAction()}
+                  >
+                    <span className='bg-gray-2 border-gray-a4 relative grid size-9 shrink-0 place-content-center overflow-hidden rounded-full border'>
+                      {hasImage ? (
+                        <img
+                          src={imageUrl}
+                          className='absolute inset-0 object-cover'
+                          alt={`Avatar for ${title}`}
+                        />
+                      ) : (
+                        <span className='text-gray-11 text-base font-medium'>
+                          {getInitials({ firstName, lastName, identifier })}
+                        </span>
+                      )}
+                    </span>
+                    <span className='flex min-w-0 flex-1 flex-col'>
+                      {title ? <span className='text-gray-12 font-medium'>{title}</span> : null}
+                      {subtitle ? (
+                        <span className='text-gray-11 w-full overflow-hidden text-ellipsis'>{subtitle}</span>
+                      ) : null}
+                    </span>
+                    <span className='text-gray-11 w-4 shrink-0 opacity-[--session-icon-opacity] transition-opacity'>
+                      <Icon.RightArrowSm />
+                    </span>
+                  </Button>
+                </li>
+              );
+            })}
+            <li className='border-gray-a4 border-t'>
+              <Button
+                type='button'
+                className={sessionAction()}
+              >
+                <span className='grid size-9 place-content-center'>
+                  <svg
+                    viewBox='0 0 36 24'
+                    className='w-9'
+                    aria-hidden
+                  >
+                    <circle
+                      cx='18'
+                      cy='12'
+                      r='11.5'
+                      strokeDasharray='2 2'
+                      className='fill-gray-2 stroke-gray-7'
+                    />
+                    <path
+                      d='M18.75 7.75a.75.75 0 1 0-1.5 0v3.5h-3.5a.75.75 0 1 0 0 1.5h3.5v3.5a.75.75 0 1 0 1.5 0v-3.5h3.5a.75.75 0 1 0 0-1.5h-3.5v-3.5Z'
+                      className='fill-gray-11'
+                    />
```

Comment:
Inlining this SVG since we have some color specific overrides for fills and strokes.